cmd/snap/cmd_userd.go

```diff
@@ -0,0 +1,133 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/godbus/dbus"
+	"github.com/godbus/dbus/introspect"
+	"github.com/jessevdk/go-flags"
+	"gopkg.in/tomb.v2"
+
+	dbusifaces "github.com/snapcore/snapd/dbus"
+	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/logger"
+)
+
+const (
+	busName  = "io.snapcraft.SafeLauncher"
+	basePath = "/io/snapcraft/SafeLauncher"
+)
+
+type registeredDBusInterface interface {
+	Name() string
+	IntrospectionData() string
+}
+
+type cmdUserd struct {
+	tomb       tomb.Tomb
+	conn       *dbus.Conn
+	dbusIfaces []registeredDBusInterface
+}
+
+var shortUserdHelp = i18n.G("Start the snap userd service")
+var longUserdHelp = i18n.G("The userd command starts the snap user session service.")
```

Comment:
s/snap//; this is really the user session service. The fact it's about snaps is implied in the context. The association in the sentence is also slightly unclear ("snap user session service" are four nouns in a row, which can be mentally aggregated in different ways.. e.g. "snap user"). Just removing the "snap" word here as well should be enough.